lib/util/subtokenize.js

```diff
@@ -122,7 +122,7 @@ function subtokenize(events) {
   return {done: !more, events: events}
 }
 
-function unravelLinkedTokens(token, context) {
+function unravelLinkedTokensImpl(token, context) {
```

Comment:
This handles a single token, maybe

```suggestion
function unravelLinkedToken(token, context) {
```

or

```suggestion
function unravelLinkedTokenImplementation(token, context) {
```

would make sense?

Reply:
agreed !